reckoner/importer.py

```diff
@@ -0,0 +1,97 @@
+# -- coding: utf-8 --
```

Comment:
?

Reply:
I copied it from elsewhere in the repo. 
It's peppy
https://www.python.org/dev/peps/pep-0263/